app/src/main/java/com/codesoom/assignment/task/controller/TaskController.java

```diff
@@ -0,0 +1,81 @@
+package com.codesoom.assignment.task.controller;
+
+import com.codesoom.assignment.task.domain.Task;
+import com.codesoom.assignment.task.domain.request.TaskRequestDto;
+import com.codesoom.assignment.task.domain.response.TaskResponseDto;
+import com.codesoom.assignment.task.service.TaskService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+import javax.validation.constraints.Min;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/tasks")
+@CrossOrigin
+@Validated
+public class TaskController {
+    private final TaskService taskService;
+
+    @Autowired
+    public TaskController(TaskService taskService) {
+        this.taskService = taskService;
+    }
+
+    @GetMapping
+    public ResponseEntity<List<TaskResponseDto>> getTasks() {
+        List<Task> tasks = taskService.getTasks();
+
+        return ResponseEntity.status(HttpStatus.OK)
+                .body(tasks.stream()
+                        .map(TaskResponseDto::from)
+                        .collect(Collectors.toList())
+                );
```

Comment:
```suggestion
        return ResponseEntity.status(HttpStatus.OK)
                .body(TaskResponseDto.from(tasks));
```

어떻게 하는지보다 무엇을 하는지가 보이게 코드를 만든다면 가독성이 좋아져요!

Reply:
그러네요!! 해당 부분을 생각하지 못했습니다..!

(6a74532) from 메서드를 오버로딩하여 수정해보았습니다!